### ISSUE TITLE
Installation of 1.5.24 with contao 4.13.* and php8.3 doesn't work

### ISSUE THREAD

**Issue by MacKP** [terminal42/contao-node#55]
Error at the Intallation:
`Fatal error: Declaration of Terminal42\NodeBundle\Picker\NodePickerProvider::supportsContext(string $context): bool must be compatible with Contao\CoreBundle\Picker\PickerProviderInterface::supportsContext($context) in /project/vendor/terminal42/contao-node/src/Picker/NodePickerProvider.php on line 50`

Quickfix:
```
"conflict": {
         "terminal42/contao-node": "1.5.24"
     },
```

Kind regards

**Comment by OMOSde** [comment]
Confirmed in one of our projects. Workaround as suggested works.

**Comment by fritzmg** [comment]
This is a regression introduced by b9d67495b2e6a05f9cdd82a3c0230a51ed0233a7

**Comment by aschempp** [comment]
Fixed in a0e2696d88d63b013b1d9b31d620a0a72a8699c6
